### PR TITLE
Add boarding passes

### PR DESCRIPTION
**Pull request by kantord**

### CODE REVIEW

File: src/booking/dataloaders/ApiSanitizer.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ export function sanitizeDetail(apiData: Object): Booking {
     assets: {
       ticketUrl: apiData.assets.eticket,
       invoiceUrl: apiData.assets.invoice,
+      boardingPasses: apiData.assets.boarding_passes,
```
[the remark this answers is not in the document]
Comment (kantord) [reply src/booking/dataloaders/ApiSanitizer.js]: hmm actually I was thinking the same, but I ended up copying the structure of the code already present there :-) Yeah I think it's a good idea to use idx in all of those nested values

File: src/booking/types/outputs/BookingAssets.js [hunk 2]
```diff
@@ -19,5 +20,16 @@ export default new GraphQLObjectType({
       description: 'URL of the invoice.',
       resolve: ({ invoiceUrl }: BookingAssets): string => invoiceUrl,
     },
+
+    boardingPasses: {
+      type: new GraphQLList(BoardingPass),
+      description: 'Boarding passes for flights in this booking',
+      resolve: ({ boardingPasses }): BookingAssets =>
+        boardingPasses &&
```
[the remark this answers is not in the document]
Comment (kantord) [reply src/booking/types/outputs/BookingAssets.js]: Good idea! I didn't think of that. Thanks :-)

File: src/booking/dataloaders/__tests__/__snapshots__/ApiSanitizer.test.js.snap [hunk 3]
```diff
@@ -42,6 +42,7 @@ Object {
     },
   },
   "assets": Object {
+    "boardingPasses": undefined,
```
[the remark this answers is not in the document]
Comment (kantord) [reply src/booking/dataloaders/__tests__/__snapshots__/ApiSanitizer.test.js.snap]: Yeah sure :-)

**Comment by kantord** [comment]
Everything should be resolved now (no pun intended)

File: src/booking/types/outputs/BookingAssets.js [hunk 4]
```diff
@@ -11,13 +12,28 @@ export default new GraphQLObjectType({
...
     },
 
     invoiceUrl: {
       type: GraphQLString,
       description: 'URL of the invoice.',
-      resolve: ({ invoiceUrl }: BookingAssets): string => invoiceUrl,
+      resolve: ({ invoiceUrl }: BookingAssets): ?string => invoiceUrl,
+    },
+
+    boardingPasses: {
```
Comment (michalsanger) [inline src/booking/types/outputs/BookingAssets.js]: Last thing I'm missing from being totally happy is having this resolver tested in GraphQL query like here for example `src/booking/queries/__tests__/Booking.baggage.test.js`

Comment (kantord) [reply src/booking/types/outputs/BookingAssets.js]: Done! :+1:  https://github.com/kiwicom/graphql/pull/238/files#diff-c0af618332f61344b386641f60aa5de4R24

File: src/booking/datasets/booking-2707251.json [hunk 5]
```diff
@@ -253,6 +253,10 @@
       "315289498": null,
       "all": null
     },
+    "boarding_passes": {
+      "315289498": null,
```
Comment (michalsanger) [inline src/booking/datasets/booking-2707251.json]: It's better to put some dummy string value then `null` so in the snapshot it's clear it works and it's not a failure.

Comment (kantord) [reply src/booking/datasets/booking-2707251.json]: :+1:

**Comment by kantord** [comment]
Should be good now? :-)

**Pull request merged**